### ISSUE TITLE
VUI concentration-Capstone instructions hard to follow

### ISSUE THREAD

**Issue by letyrodridc** [udacity/aind-issue-reports#199]
By student Michael Gamba:
Instructions for setting up the GPU are still wrong at every turn. The link to aws instructions in the Github repo points to something else. The tensorflow installation instructions don't work. The GPU related instructions point to tensorflow docs, which are completely misleading since the deep learning ami's already have all of the software installed and registered.
Link to instructions:
https://github.com/udacity/AIND-VUI-Capstone

**Comment by cgearhart** [comment]
Internal ticket #J1913
